sklearn_numba_dpex/common/tests/test_kernels.py

```diff
@@ -6,11 +8,12 @@
 
 from sklearn_numba_dpex.common.kernels import (
     make_argmin_reduction_1d_kernel,
-    make_sum_reduction_2d_axis1_kernel,
+    make_sum_reduction_2d_kernel,
 )
 from sklearn_numba_dpex.testing.config import float_dtype_params
 
 
+@pytest.mark.parametrize("axis", [0, 1])
```

Comment:
I guess `axis` and `expected_result` are coupled and better be joined in the same parametrization mark, right?

Reply:
Yes, but for some cases, reducing on another axis should given different results, right?

For instance, the first third parametrization case for `array_in, expected_result` must when `axis==0` be:

```
array([15.0, 18.0, 21.0, 24.0, 27.0])
```

right?

Reply:
No the test wasn't supposed to be read like this.

But since you and @ogrisel had the same remark I ended up rewriting it, hope that can solve the confusion about axis and expected values.